oap-server/server-query-plugin/prometheus-query-plugin/src/main/java/org/apache/skywalking/oap/query/prometheus/entity/TimeValuePair.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.query.prometheus.entity;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Data;
+import org.apache.skywalking.oap.query.prometheus.entity.codec.TimeValuePairSerializer;
+
+@Data
+@JsonSerialize(using = TimeValuePairSerializer.class)
+public class TimeValuePair {
+    private final long time;
+    private String value;
```

Comment:
Looks like this class is a value holder and can be an immutable class

```suggestion
    private final String value;
```